contracts/StudioDAOFundingCycleDataSource.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.6;
+
+import './interfaces/IJBFundingCycleDataSource.sol';
+
+contract StudioDAOFundingCycleDataSource is IJBFundingCycleDataSource {
```

Comment:
You need a data source which points to the didPay or redeem (here redeem is not used)